packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,230 @@
+import { Badge } from 'terra-clinical-item-view/package.json?dev-site-package';
+import { Notice } from "@cerner/terra-docs";
+import ItemViewWithHeader from '../example/ItemViewWithHeader?dev-site-example';
+import ItemViewTwoColumnWithHeader from '../example/ItemViewTwoColumnWithHeader?dev-site-example';
+
+<Badge />
+
+# Accessibility Guide for Terra Clinical Item View
+
+## Why is this important?
+
+Terra Clinical Item View allows you to create a view filled with information in the form of Clinical Item Displays. Improper usage of this component may prevent end users from understanding content and necessary context to interact with the page. 
+
+## Accessibility Considerations
+
+#### General notes
+- **Related information should be presented as close to each other as possible.** It is best practice to keep related information above or below its related counterparts.
+  - For example - presenting related information across from each other in the two column view can make it more difficult for users to understand that the information is related.
+Reading by column from top to bottom is the more common way information is consumed visually, so when presenting this information by row between the two columns it might be confusing for both sighted and non-sighted users.
+- When passing in displays, keep in mind that only 8 can be used within a view at a time. Any additional displays passed in will be ignored.
+- Displays are also split between two columns with every other display being moved to the right hand column.
+
+----
+
+### Code Considerations
+
+#### Headings
+
+<Notice variant="important" ariaLevel="3">
```

Comment:
Notice blocks should _not_ span multiple paragraphs. it They should be limited to calling out a single important piece of information. In this case, it seems like this paragraph alone should be in the notice block. Everything else should be regular text.

<img width="1118" alt="CleanShot 2023-09-21 at 17 09 48@2x" src="https://github.com/cerner/terra-clinical/assets/38024451/863560b4-cdae-4243-9c63-d07d24b3522d">

Reply:
Changed but I kept the first line in the notice block - is that ok with you? I thought it was important enough to include: https://github.com/cerner/terra-clinical/pull/916/commits/0fc54b77d974702f4c6a920ad2325445563439e6

Reply:
That looks fine!